keras/models.py

```diff
@@ -175,6 +175,15 @@ def get_function_name(o):
         return o.__name__
 
 
+def accuracy_metric(y, y_gold, class_mode):
```

Comment:
The variable naming convention used elsewhere in the code base is `y_pred`, `y_true`.